pandas/core/frame.py

```diff
@@ -4070,7 +4070,11 @@ def _apply_standard(self, func, axis, ignore_failures=False, reduce=True):
         # e.g. if we want to apply to a SparseFrame, then can't directly reduce
         if reduce:
 
-            values = self.values
+            # The 'values' attribute of a pandas Series
+            # is a numpy ndarray, so the dtype will be
+            # guaranteed to be valid when passed into
```

Comment:
It will not help in this case because this logic <a href="https://github.com/pydata/pandas/blob/e9558d3b926e089dcda4ce4ff38f1e980777424b/pandas/core/series.py#L2906">here</a> IINM will just return the same data structure with the exact same data type that we are trying to avoid in the context of the bug.